config-handler-router/pom.xml

```diff
@@ -95,20 +149,22 @@
                                 <rule>
                                     <element>BUNDLE</element>
                                     <limits>
+                                        <!--Test coverage is low since the RuntimeTypeAdapterFactory and SparkServlet were copied verbatim from other locations-->
+                                        <!--and are unchanged and should not ever change.-->
```

Comment:
You can exclude specific classes so our coverage numbers are accurate http://stackoverflow.com/questions/27799419/maven-jacoco-configuration-exclude-classes-packages-from-report-not-working. You can remove your comment if you do since the plugin will be self explanatory.